src/pages/StudentExchange/StudentExchange.scss

```diff
@@ -0,0 +1,40 @@
+.studentexchange {
+  &__title {
+
+    line-height: 1.2;
+
+    @media screen and (max-width: $breakpoint-laptop) {
+      text-align: center;
+    }
+    
+    @media screen and (max-width: $breakpoint-mobile) {
+      text-align: center
+    }
+  }
+
+  &__description {
+    @media screen and (max-width: $breakpoint-laptop) {
+      margin: 3rem 2rem;
+    }
+
+    @media screen and (max-width: $breakpoint-tablet-large) {
+      margin: 3rem 2rem;
+    }
+
+    @media screen and (max-width: $breakpoint-tablet) {
+      margin: 3rem 2rem;
+    }
+    
+    @media screen and (max-width: $breakpoint-mobile-large) {
+      margin: 3rem 2rem;
+    }
+
+    @media screen and (max-width: $breakpoint-mobile) {
+      margin: 3rem 2rem;
+    }
+  }
+}
+
+ol {
```

Comment:
This is going to override the global style, is that what you want? I would recommend nesting in the parent style